src/modules/map/components/footer/backgroundSelector/BackgroundSelectorSquared.vue

```diff
@@ -82,7 +83,7 @@ const { show, animate, getImageForBackgroundLayer, toggleShowSelector, onSelectB
 @import './bg-selector';
 
 $main-element: '.bg-selector-squared';
-$square-button-width: 7rem;
```

Comment:
Just wonder if using `6rem` or `5rem` would have achieved to get close to the old viewer size of `98px` ?

Reply:
I tried to play a little bit with this value, and at `96px` there would be some line-break in English... So I prefered having some hard values that we are sure won't produce this...